src/Microsoft.SqlTools.ServiceLayer/Utility/DatabaseUtils.cs

```diff
@@ -245,5 +246,36 @@ private static void SpecialPostDropActionsForObject(CDataContainer dataContainer
                 dataContainer.Server.ResourceGovernor.Alter();
             }
         }
+
+        public static string[] LoadSqlLogins(ServerConnection serverConnection)
+        {
+            return LoadItems(serverConnection, "Server/Login");
+        }
+
+        public static string[] LoadItems(ServerConnection serverConnection, string urn)
+        {
+            List<string> items = new List<string>();
+            Request req = new Request();
+            req.Urn = urn;
+            req.ResultType = ResultType.IDataReader;
+            req.Fields = new string[] { "Name" };
+
+            Enumerator en = new Enumerator();
+            using (IDataReader reader = en.Process(serverConnection, req).Data as IDataReader)
+            {
+                if (reader != null)
+                {
+                    string name;
+                    while (reader.Read())
+                    {
+                        // Get the permission name
+                        name = reader.GetString(0);
+                        items.Add(name);
+                    }
+                }
+            }
+            items.Sort();
```

Comment:
this line is the only functional change.  Rest of PR is moving these methods to a better location.